cibuildwheel/util.py

```diff
@@ -71,6 +71,15 @@
 
 BuildFrontend = Literal["pip", "build"]
 
+
+def build_frontend_or_default(
+    setting: BuildFrontend | Literal["default"], default: BuildFrontend
+) -> BuildFrontend:
+    if setting == "default":
+        return default
+    return setting
+
+
```

Comment:
minor detail, if switching the default from pip to build in the future, we'd now have to change multiple files to ensure consistency.
I wonder if taking the `config.identifier` rather than a `default` BuildFrontend and managing the actual default BuildFrontend in this function would be better or worse ?

Reply:
What about just giving the default argument a default value `default = "pip"`?